source/wp-content/themes/wporg-gutenberg/functions.php

```diff
@@ -37,253 +37,258 @@ function () {
+				'allowedBlockTypes'      => $allowed_block_types,
+				'disableCustomColors'    => get_theme_support( 'disable-custom-colors' ),
+				'disableCustomFontSizes' => get_theme_support( 'disable-custom-font-sizes' ),
+				'disablePostFormats'     => ! current_theme_supports( 'post-formats' ),
+				'titlePlaceholder'       => apply_filters( 'enter_title_here', __( 'Add title', 'gutenberg' ), $post ),
+				'bodyPlaceholder'        => apply_filters( 'write_your_story', __( 'Start writing or type / to choose a block', 'gutenberg' ), $post ),
+				'isRTL'                  => is_rtl(),
+				'autosaveInterval'       => 10,
+				'maxUploadFileSize'      => $max_upload_size,
+				'allowedMimeTypes'       => get_allowed_mime_types(),
+				'styles'                 => $styles,
+				'imageSizes'             => gutenberg_get_available_image_sizes(),
+				'richEditingEnabled'     => user_can_richedit(),
+				'fullscreenMode'         => true,
+
+				// Ideally, we'd remove this and rely on a REST API endpoint.
+				'postLock'               => $lock_details,
+				'postLockUtils'          => array(
+					'nonce'       => wp_create_nonce( 'lock-post_' . $post->ID ),
+					'unlockNonce' => wp_create_nonce( 'update-post_' . $post->ID ),
+					'ajaxUrl'     => admin_url( 'admin-ajax.php' ),
+				),
+
+				// Whether or not to load the 'postcustom' meta box is stored as a user meta
+				// field so that we're not always loading its assets.
+				'enableCustomFields'     => (bool) get_user_meta( get_current_user_id(), 'enable_custom_fields', true ),
 			);
-		}
 
-		if ( ! empty( $post_type_object->template ) ) {
-			$editor_settings['template']     = $post_type_object->template;
-			$editor_settings['templateLock'] = ! empty( $post_type_object->template_lock ) ? $post_type_object->template_lock : false;
-		}
+			$post_autosave = gutenberg_get_autosave_newer_than_post_save( $post );
+			if ( $post_autosave ) {
+				$editor_settings['autosave'] = array(
+					'editLink' => get_edit_post_link( $post_autosave->ID ),
+				);
+			}
+
+			if ( ! empty( $post_type_object->template ) ) {
+				$editor_settings['template']     = $post_type_object->template;
+				$editor_settings['templateLock'] = ! empty( $post_type_object->template_lock ) ? $post_type_object->template_lock : false;
+			}
 
-		$editor_context  = new WP_Block_Editor_Context( array( 'post' => $post ) );
-		$editor_settings = get_block_editor_settings( $editor_settings, $editor_context );
+			$editor_context  = new WP_Block_Editor_Context( array( 'post' => $post ) );
+			$editor_settings = get_block_editor_settings( $editor_settings, $editor_context );
 
-		$init_script = <<<JS
-( function() {
-	window._wpLoadBlockEditor = new Promise( function( resolve ) {
-		wp.domReady( function() {
-			resolve( wp.editPost.initializeEditor( 'editor', "%s", %d, %s, %s ) );
+			$init_script = <<<JS
+	( function() {
+		window._wpLoadBlockEditor = new Promise( function( resolve ) {
```

Comment:
Does this need to be loaded on mobile?.. along with the `$script` block below...

Reply:
I don't think they are loaded on mobile. (or you're saying that they should be loaded?)

I think it's the format making it hard to read, and the JS part is the one making it unclear (You can see that [originally](https://github.com/WordPress/wporg-gutenberg/blob/trunk/source/wp-content/themes/wporg-gutenberg/functions.php#L270-L277) it is aligned to the left side without the indent,  and I thought it's might made on purpose at first so I didn't add one more indent when including `else` into the logic). I'll add one more indent first to make the format clearer.

Reply:
Ah I see, yeah my code collapse function got confused. Thanks. No changes needed.

Reply:
Would the extra indent affect anything? https://github.com/WordPress/wporg-gutenberg/pull/47/commits/da207bb814e2f6ae3cd0bba5e11b1d72aefc2d29 If not, I'm tempted to keep it as the format looks more readable.